pkg/meta/store.go

```diff
@@ -248,6 +248,10 @@ func (s *Store) GetWithPrefix(prefix string) ([]Object, error) {
 func (s *Store) KeysWithPrefix(prefix string) ([]string, error) {
 	var keys []string
 
+	if len(prefix) == 0 {
+		return keys, nil
```

Comment:
I suggest that here we return an error, like `return nil, fmt.Errorf("input prefix cannot be empty when getting object in store")`